### PR TITLE
Postman tests

### PR DESCRIPTION
**Pull request by chsergey**
Markdown-syntax file with Postman tests wiil be parsed and test will be added to result collection.json file

### CODE REVIEW

**Comment by kielabokkie** [comment]
Thanks for this PR @chsergey. I've been really busy lately so haven't had the chance to look at it in details yet. Will do soon as I think it's a great added feature!

**Comment by chsergey** [comment]
thanks for Blueman, @kielabokkie =) 
awaiting for review and merge)

File: src/Blueman/Console/Command/ConvertCommand.php [hunk 1]
```diff
@@ -39,6 +39,20 @@ protected function configure()
...
+                'tests-filename',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'The JSON file name with Postman tests (located at --path)',
+                'blueman.tests.md'
+            )
+            ->addOption(
+                'tests-include',
+                null,
+                InputOption::VALUE_OPTIONAL,
```
Comment (kielabokkie) [inline src/Blueman/Console/Command/ConvertCommand.php]: Can you make this `InputOption::VALUE_NONE` and change the option name to `include-tests` that way the command will be simpler, e.g. `$ blueman convert api.json --include-tests  --tests-filename=<YOUR_FILNAME>.md`

File: src/Blueman/Console/Command/ConvertCommand.php [hunk 2]
```diff
@@ -65,10 +81,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
             );
         }
 
+        $tests = array();
+        if('true' === $input->getOption('tests-include')) {
```
Comment (kielabokkie) [inline src/Blueman/Console/Command/ConvertCommand.php]: Can you swap the order to match the coding style of the rest of the code? e.g. `if ($input->getOption('tests-include') === true) {`

File: src/Blueman/Console/Command/ConvertCommand.php [hunk 3]
```diff
@@ -65,10 +81,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+
+            $testsFile = $filePath . $input->getOption('tests-filename');
+
+            if($testsFileExists = file_exists($testsFile)) {
```
Comment (kielabokkie) [inline src/Blueman/Console/Command/ConvertCommand.php]: What is the variable `testsFileExists` doing here? It's not being used anywhere else. If you just want to check if the test file exists then do `if (file_exists($testsFile) === true) {` instead.

File: composer.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "symfony/console": "~2.4",
-        "rhumsaa/uuid": "~2.7"
+        "ramsey/uuid": "~2.7"
```
Comment (kielabokkie) [inline composer.json]: You changed to the newer version of the uuid package in the composer file but the use statement in the `ConvertCommand.php` file is unchanged. Is this correct?

Comment (chsergey) [inline composer.json]: yes. in uuid v2.7.4 namespace is Rhumsaa\Uuid

File: src/Blueman/Console/Command/ConvertCommand.php (hunk 3 above)
Comment (chsergey) [inline src/Blueman/Console/Command/ConvertCommand.php]: you right. I forgot remove this variable due dev-process

File: src/Blueman/Console/Command/ConvertCommand.php (hunk 2 above)
Comment (chsergey) [inline src/Blueman/Console/Command/ConvertCommand.php]: ok

File: src/Blueman/Console/Command/ConvertCommand.php (hunk 1 above)
Comment (chsergey) [inline src/Blueman/Console/Command/ConvertCommand.php]: good idea)

**Comment by chsergey** [comment]
@kielabokkie, code fixes done

**Comment by kielabokkie** [comment]
Thanks @chsergey! I'll merge this PR in the next few days and create a new release. I'll let you know once it is done.

**Comment by chsergey** [comment]
As you like, @kielabokkie)

**Comment by kielabokkie** [comment]
This is merged and I just released a new version. Run `blueman self-update` to update do the latest version (v1.3.0). Thanks @chsergey!

**Pull request merged**